bedrock/base/templates/includes/structured-data/organization/mozilla-corporation-organisation.json

```
@@ -0,0 +1,460 @@
+{
+    "@context": "http://schema.org",
+    "@type": "Organization",
+    "@id": "https://www.mozilla.org/#organization",
+    "name": "Mozilla",
+    "alternateName": "MoCo",
+    "legalName": "Mozilla Corporation",
+    "description": "Mozilla makes browsers, apps, code and tools that put people before profit. Our mission: Keep the internet open and accessible to all.",
+    "url": "https://www.mozilla.org",
+    "logo": "https://www.mozilla.org/media/img/mozorg/mozilla-256.4720741d4108.jpg",
```

We shouldn't be using hashed filenames like this in our JSON. If this image file gets updated, the URL will 404.

Instead I'd suggest using the static image helper, like we do in templates.

```suggestion
    "logo": "{% filter absolute_url %}{{ static('img/mozorg/mozilla-256.jpg') }}{% endfilter %}",
```